bcipy/bci_tasks/rsvp/calibration.py

```diff
@@ -102,7 +102,19 @@ def execute(self):
                 if not get_user_input(self.rsvp, self.wait_screen_message,
                                       self.wait_screen_message_color):
                     break
-
+                
+                #Take a break every number of trials defined in parameters.json
+                if not self.parameters['trials_before_break'] == 0:
+                    if (not idx_o == 0) and ((idx_o % self.parameters['trials_before_break']) == 0):
```

Comment:
This might be easier to read if it was `if (idx_o % self.trials_before_break) == 0 and idx_o != 0`.  Be sure to add a comment above it to explain the logic